### PR TITLE
changelog update for 1.5.7

### PR DESCRIPTION
**Pull request by meln1k**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+## Changes from 1.5.6 to 1.5.7
+Bugfix release
```
Comment (ichernetsky) [inline changelog.md]: I think it is not a bugfix release anymore, because preferential GPU scheduling is included.

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+## Changes from 1.5.6 to 1.5.7
+### Fixed issues
+
+- [MARATHON-8039](https://jira.mesosphere.com/browse/MARATHON-8039) Complete deployment after it was removed to avoid race condition. (#5944)
+- [COPS-2224](https://jira.mesosphere.com/browse/COPS-2224) Optimize `v2/groups` request performance (#6001)
+- [MARATHON-8112](https://jira.mesosphere.com/browse/MARATHON-8112) Open the persistence store before taking a backup using CLI (#6057)
+- [MARATHON-7751](https://jira.mesosphere.com/browse/MARATHON-7751) Do not start more instances than needed (#6054) 
+- [MARATHON-8124](https://jira.mesosphere.com/browse/MARATHON-8124) Always unreserve resources for non-existing instances (#6072)
+- [MARATHON-7940](https://jira.mesosphere.com/browse/MARATHON-7940) Disable Connection Pooling for HTTP Health Checks (#6098) 
+- Many documentation fixes
```
Comment (kensipe) [inline changelog.md]: minor... how about `- Improved Documentation`

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
+## Changes from 1.5.6 to 1.5.7
+### Fixed issues
+
+- [MARATHON-8039](https://jira.mesosphere.com/browse/MARATHON-8039) Complete deployment after it was removed to avoid race condition. (#5944)
+- [COPS-2224](https://jira.mesosphere.com/browse/COPS-2224) Optimize `v2/groups` request performance (#6001)
```
Comment (kensipe) [inline changelog.md]: Change out COPS (private ticket) for a public ticket

**Comment by kensipe** [comment]
added last comment in case you missed the slack chat

**Pull request merged**